upup/pkg/fi/cloudup/azuretasks/disk_test.go

```diff
@@ -40,7 +41,8 @@ func newTestDisk() *Disk {
 		ResourceGroup: &ResourceGroup{
 			Name: to.Ptr("rg"),
 		},
-		SizeGB: to.Ptr[int32](32),
+		SizeGB:     to.Ptr[int32](32),
+		VolumeType: to.Ptr(testVolumeType),
```

Comment:
I don't think there's much value in extracting this as a constant.
```suggestion
		VolumeType: to.Ptr(compute.DiskStorageAccountTypesStandardSSDLRS),
```